specification/07-theory_operations.adoc

```diff
@@ -431,4 +431,98 @@ TSM ->> VMM: [COVH] - spdm_covh_connect_device()
+TCB, and to explicitly notify both the TSM and the host supervisor domain
+manager about its decision. The TVM should check for the following security
+attributes before being able to decide whether or not it can safely add a TDI
+into its TCB:
+
+1. **SPDM session establishment**: A secured SPDM session must be established
+   between the TDI’s DSM and the TSM. TVM verifies that attribute from the TSM,
+   through the `sbi_covg_get_device_link()` `COVG` ABI.
+2. **IDE link**: The PCIe physical link between the Root Port and the physical
+   device must be confidentiality and integrity protected through IDE. As for
+   the SPDM session, the TVM calls into the `sbi_covg_get_device_link()` `COVG`
+   ABI to verify that attribute from the TSM.
+3. **Firmware update policy**: The TVM must verify that the device firmware
+   update security attributes are acceptable, by calling into
+   `sbi_covg_get_device_update_attrs()`. The returned set of attributes,
+   including the firmware update policy, lets the TVM check if runtime firmware
+   update is allowed for the device, and under which conditions. Depending on
+   the TVM’s own policy, this may be acceptable or not.
+4. **TDI state**: Before accepting a TDI into its TCB, a TVM must verify that
+   its configuration is immutable, and in particular that the host can not
+   modify it without having all in-flight transactions being discarded. TEE-IO
+   capable physical devices follow the TDISP specification and can guarantee
+   that immutability state once the TDI has been moved to the TDISP LOCKED
+   state. The transition from TDISP `UNLOCKED` to `LOCKED`, although triggered
+   by the host supervisor domain manager, is done by the TSM. As such, the TVM
+   can query the TSM for the TDI state through the
+   `sbi_covg_get_interface_state()` `COVG` ABI. A TVM must not accept a TDI if
+   it’s in any other TDISP state than LOCKED.
+5. **TDI identity**: Verifying that the TDI is in an immutable state across a
+   secured SPDM and physical link is necessary but the TVM must also verify
+   the TDI identity in order to decide if it can accept it into its TCB:
+   a. First the TVM must first verify the authenticity of the TDI by getting
+      its certificate chain from the TSM, through the
+      `sbi_covg_get_device_certificate()` `COVG` ABI. The TVM should then verify
+      the chain against a provisioned and measured trust anchor list.
+   b. Once the TDI certificate authenticity is verified, the TVM must then
+      challenge the TDI by having it sign a piece of data, making sure that the
+      TDI actually owns the private key bound to its certificate. This is
+      achieved by getting the TDI measurements from the TSM through the
+      `sbi_covg_get_device_measurement()` `COVG` ABI. This gives a TDI signed set
+      of measurements that the TVM can verify with the previously acquired TDI
+      certificate.
+6. **TDI trustworthiness**: During step 5, the TVM fetched the TDI signed
```

Comment:
Suggest to use `TDI measurement` to make it explicit.

I am confused to see "trustworthiness" - not sure what it is.

Reply:
Here I meant that the TVM is responsible for attestating of the TDI trustworthiness.

Reply:
If it is for attestation, then TDI identity should be merged into TDI trustworthiness.

I am OK to say "TDI trustworthiness" alone, including "TDI identity" + "TDI measurement". 
Or call "TDI identity" + "TDI measurement" explicitly.

Reply:
"Device trustworthiness", yes.

Reply:
I combined the identiy, challenge, and attestation parts under a `Device trustworthiness`section. I think it addresses your comments here.